<launch>

  <node pkg="rbx1_vision" name="ros2opencv2" type="ros2opencv2.py" output="screen">
    <remap from="input_rgb_image" to="/camera/rgb/image_raw" />
    <remap from="input_depth_image" to="/camera/depth/image_raw" />
  </node>
  
</launch>
